### PR TITLE
[ICP-12259] Check interval change for Fibaro CO Sensor

### PR DESCRIPTION
**Pull request by PKacprowiczS**
@greens @kianooshST @KKlimczukS @MGoralczykS @ZWozniakS

### CODE REVIEW

File: devicetypes/fibargroup/fibaro-co-sensor-zw5.src/fibaro-co-sensor-zw5.groovy [hunk 1]
```diff
@@ -92,7 +92,7 @@ metadata {
 }
 
 def installed() {
-	sendEvent(name: "checkInterval", value: 86520, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	sendEvent(name: "checkInterval", value: 43632, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```
Comment (kianooshkarami) [inline devicetypes/fibargroup/fibaro-co-sensor-zw5.src/fibaro-co-sensor-zw5.groovy]: I always end up doing the math to figure out how long that really is, could you do something like `12 * 60 * 60  + whatever`

**Review by cbaumler — COMMENTED** [review]
Just curious, what's the rationale/backstory for this change?

**Comment by PKacprowiczS** [comment]
@cbaumler Cert team was concerned that check interval is too long (it was equal 24h), and since device itself automatically checks in every 6 hours, it even makes more sense to shorten check interval to 12h.

**Review by cbaumler — APPROVED** [review]
LGTM

**Review by greens — APPROVED** [review]

**Pull request merged**